SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||figma.com/api/web_logger/*
```

Comment:
`*` is not necessary here, but we use `*` at the end when we want to avoid using regex like this `/web_logger/`.